### ISSUE TITLE
Printing too many output lines per epoch

### ISSUE THREAD

**Issue by zhanyc10** [fchollet/keras#5224]
Hello,

I just installed Keras with Theano backend.  I am using the example code from
https://github.com/fchollet/keras/blob/master/examples/imdb_lstm.py

The program can run but print so many lines per epoch:

Using Theano backend.
WARNING (theano.sandbox.cuda): The cuda backend is deprecated and will be removed in the next release.  Please switch to the gpuarray backend. You can get more information about how to switch at this URL:
 https://github.com/Theano/Theano/wiki/Converting-to-the-new-gpu-back-end%28gpuarray%29

Using gpu device 0: GeForce GTX 1060 6GB (CNMeM is enabled with initial size: 82.5% of memory, cuDNN 5110)
Loading data...
25000 train sequences
25000 test sequences
Pad sequences (samples x time)
X_train shape: (25000L, 80L)
X_test shape: (25000L, 80L)
Build model...
Train...
Train on 25000 samples, validate on 25000 samples
Epoch 1/15
   32/25000 [..............................] - ETA: 78s - loss: 0.6960 - acc: 0.4062
   64/25000 [..............................] - ETA: 78s - loss: 0.6911 - acc: 0.4844
   96/25000 [..............................] - ETA: 76s - loss: 0.6895 - acc: 0.5312
  128/25000 [..............................] - ETA: 73s - loss: 0.6862 - acc: 0.5703
  160/25000 [..............................] - ETA: 79s - loss: 0.6903 - acc: 0.5563
  192/25000 [..............................] - ETA: 77s - loss: 0.6834 - acc: 0.5833
  224/25000 [..............................] - ETA: 76s - loss: 0.6927 - acc: 0.5491
  256/25000 [..............................] - ETA: 78s - loss: 0.6915 - acc: 0.5469
  288/25000 [..............................] - ETA: 77s - loss: 0.6915 - acc: 0.5486
  320/25000 [..............................] - ETA: 79s - loss: 0.6966 - acc: 0.5219
  352/25000 [..............................] - ETA: 78s - loss: 0.6984 - acc: 0.5057
  384/25000 [..............................] - ETA: 78s - loss: 0.6976 - acc: 0.5026
......
24832/25000 [============================>.] - ETA: 0s - loss: 0.5113 - acc: 0.7486
24864/25000 [============================>.] - ETA: 0s - loss: 0.5112 - acc: 0.7486
24896/25000 [============================>.] - ETA: 0s - loss: 0.5112 - acc: 0.7486
24928/25000 [============================>.] - ETA: 0s - loss: 0.5110 - acc: 0.7488
24960/25000 [============================>.] - ETA: 0s - loss: 0.5110 - acc: 0.7490
24992/25000 [============================>.] - ETA: 0s - loss: 0.5109 - acc: 0.7490
25000/25000 [==============================] - 101s - loss: 0.5109 - acc: 0.7491 - val_loss: 0.4067 - val_acc: 0.8239
Epoch 2/15
   32/25000 [..............................] - ETA: 74s - loss: 0.1730 - acc: 0.9688
   64/25000 [..............................] - ETA: 71s - loss: 0.3168 - acc: 0.8594
   96/25000 [..............................] - ETA: 70s - loss: 0.3654 - acc: 0.8229
......

Is it normal? How can I fix it? Thanks!

**Comment by tpinetz** [comment]
Yes you can use different verbose settings in your model.fit method (https://keras.io/models/model/#fit).

**Comment by zhanyc10** [comment]
@tpinetz Thank you for the suggestion.

Actually I tried these verbose settings but it did not work.

**Comment by jonimatix** [comment]
Is there a final resolution to this please? What is the suggested action?

Thank you.
